README.md

```diff
@@ -1,58 +1,44 @@
 
-# AssemblyScript
+You can run all these tests with one command:
 
-_Using Gitpod? You can skip these setup steps!_
+```bash
+cargo test
+```
 
-To run this project locally:
+If you want to run only simulation tests, you can use `cargo test simulation`, since all the simulation tests include "simulation" in their names.
 
-1. Prerequisites: Make sure you have Node.js ≥ 12 installed (https://nodejs.org), then use it to install [yarn]: `npm install --global yarn` (or just `npm i -g yarn`)
-2. Install dependencies: `yarn install` (or just `yarn`)
 
-Now you can run all the [AssemblyScript]-related scripts listed in `package.json`! Scripts you might want to start with:
+## Notes
 
-- `yarn test:unit:as`: Runs all AssemblyScript tests with filenames ending in
-  `unit.spec`
-- `yarn build:as`: Compiles the AssemblyScript contracts to [Wasm] binaries
+ - The maximum balance value is limited by U128 (`2**128 - 1`).
+ - JSON calls should pass U128 as a base-10 string. E.g. "100".
+ - This does not include escrow functionality, as `ft_transfer_call` provides a superior approach. An escrow system can, of course, be added as a separate contract or additional functionality within this contract.
 
-## Data collection
+## No AssemblyScript?
 
-By using Gitpod in this project, you agree to opt-in to basic, anonymous analytics. No personal information is transmitted. Instead, these usage statistics aid in discovering potential bugs and user flow information.
+[near-contract-standards] is currently Rust-only. We strongly suggest using this library to create your own Fungible Token contract to ensure it works as expected.
 
-  [rust]: https://www.rust-lang.org/
-  [yarn]: https://yarnpkg.com/
-  [AssemblyScript]: https://assemblyscript.org/
-  [Wasm]: https://webassembly.org/
-  
-
-# Course on FTs on NEAR
-
-If you want to learn more about fungible tokens and create your own project, head over to the course on ["Building and Issuing Fungible Tokens on NEAR"](https://vitalpoint.ai/academy/building-and-issuing-fungible-tokens-on-near/). The course was created by [Vital Point AI](https://github.com/ALuhning), who are part of the [NEAR Guilds Program](https://near.org/guilds/)
-
-What you will learn
-* Fundamentals of FTs
-* NEAR enhancement proposals (NEPs) vs Ethereum Improvement Proposals (EIPs)
-* How to consult the relevant standards and specifications - specifically the NEAR Fungible Token Standard (NEP-21) and Ethereum ERC-20
-* Build a robust AssemblyScript contract (and supporting contracts) that issues and manages a Fungible Token on the NEAR blockchain
-* with a React front-end
+Someday NEAR core or community contributors may provide a similar library for AssemblyScript, at which point this example will be updated to include both a Rust and AssemblyScript version.
```

Comment:
Does this "No AssemblyScript?" section seem accurate?

Reply:
Yeah, I'm down with this. We could create a bounty if folks want an AS version, but I like the wording here. Thank you!